config/manager/manager.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         args:
         - "--leader-elect"
         - "--metrics-bind-addr=localhost:8080"
-        - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},ClusterResourceSet=${EXP_CLUSTER_RESOURCE_SET:=false},ClusterTopology=${CLUSTER_TOPOLOGY:=false}"
+        - "--feature-gates=MachinePool=${EXP_MACHINE_POOL:=false},ClusterResourceSet=${EXP_CLUSTER_RESOURCE_SET:=false},ClusterTopology=${CLUSTER_TOPOLOGY:=false},RuntimeExtension=${EXP_RUNTIME_EXTENSION:=false}"
```

Comment:
Let's use RuntimeSDK as name for the feature

Reply:
Fine for now - it's weird to me to have this feature flag be "SDK" but any name will do :slightly_smiling_face:

Reply:
Yup. Potentially we rename against CAPI main, but we renamed the proposal to RuntimeSDK and a Runtime Extension nowadays is just the implementation of a RuntimeHook and not the headline anymore